src/targeting/CombatTargeting.ts

```diff
@@ -60,22 +61,35 @@ export class CombatTargeting {
 		return maxBy(targets, function(hostile) {
 			if (hostile.hitsPredicted == undefined) hostile.hitsPredicted = hostile.hits;
 			if (hostile.pos.lookForStructure(STRUCTURE_RAMPART)) return false;
+			if (hostile.name.substring(hostile.name.indexOf('_')) == 'ANTITIGGA') {
+				console.log('TIGGA ATTACK! :D');
+				return hostile.pos.findInRange(FIND_MY_CREEPS, 2).length;
+				// return (hostile.pos.getRangeTo(this.colony.bunker.anchor) <= 6 + 2) || hostile.hitsMax - hostile.hitsPredicted;
+			}
 			return hostile.hitsMax - hostile.hitsPredicted
 				   + CombatIntel.getHealPotential(hostile) + (CombatIntel.towerDamageAtPos(hostile.pos) || 0);
 		});
 	}
 
-	static findClosestHostile(zerg: Zerg, checkReachable = false, ignoreCreepsAtEdge = true): Creep | undefined {
+	static findClosestHostile(zerg: Zerg, checkReachable = false, ignoreCreepsAtEdge = true, playerOnly = false,
```

Comment:
something that takes this many optional arguemnts should have an `opts` class passed in, pls modify accordingly